src/django_pg_migration_tools/operations.py

```diff
@@ -5,9 +5,18 @@
 from django.contrib.postgres import operations as psql_operations
 from django.db import migrations, models
 from django.db.backends.base import schema as base_schema
+from django.db.migrations.operations import base as base_operations
+from django.db.migrations.operations import models as operation_models
 
 
-class BaseIndexOperation:
+# Postgres doesn't allow any relation names that are longer than 63 bytes.
+MAX_RELATION_NAME_LEN = 63
```

Comment:
This constant is now no longer used, and can be omitted.

Reply:
Good catch: 3621d43efd65e21f65f4c14de4f3a17d7cc948fc